cli/cmd/config.go

```diff
@@ -35,6 +36,7 @@ func init() {
 	configListCmd.Flags().BoolVarP(&reveal, "reveal", "r", false, "Print the actual app config values instead of masking them")
 
 	configCmd.AddCommand(configGetCmd)
+	configGetCmd.Flags().StringVar(&getKey, "get-key", "", "Specify the key that you want to get")
```

Comment:
hmm this doesn't seem correct. I think the issue is in `var configGetCmd = &cobra.Command{` subcommand below (line149). I don't think its validating the correct number of args that are passed to `args`. When you fix this, can you add reproduction steps to your description?